op-deployer/pkg/deployer/state/chain_intent.go

```diff
@@ -40,13 +40,16 @@ type ChainIntent struct {
 	BaseFeeVaultRecipient      common.Address            `json:"baseFeeVaultRecipient" toml:"baseFeeVaultRecipient"`
 	L1FeeVaultRecipient        common.Address            `json:"l1FeeVaultRecipient" toml:"l1FeeVaultRecipient"`
 	SequencerFeeVaultRecipient common.Address            `json:"sequencerFeeVaultRecipient" toml:"sequencerFeeVaultRecipient"`
+	OperatorFeeVaultRecipient  common.Address            `json:"operatorFeeVaultRecipient" toml:"operatorFeeVaultRecipient"`
```

Comment:
Should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240